src/providers/wms/qgstilescalewidget.cpp

```diff
@@ -105,6 +105,9 @@ void QgsTileScaleWidget::scaleChanged( double scale )
   mSlider->blockSignals( true );
   mSlider->setValue( i );
   mSlider->blockSignals( false );
+
+
+  mLabelScale->setText( QStringLiteral( "Zoom: %1" ).arg( i ) );
```

Comment:
```suggestion
  mLabelScale->setText( tr( "Zoom: %1" ).arg( i ) );
```

Reply:
@nyalldawson ah yeah fair enough because it needs to be translated. Forgot about that.